modules/tinymce/src/plugins/table/test/ts/browser/InsertColumnTableResizeTest.ts

```diff
@@ -1,200 +1,455 @@
-                '<td></td>' +
-              '</tr>' +
-            '</tbody>' +
-          '</table>'
-  };
+describe('browser.tinymce.plugins.table.InsertColumnTableResizeTest', () => {
+  /**
+   * Test the width of tables before and after column operations
+   * There is a 2 pixel delta allowed from the expected width, to account for table borders
+   */
 
-  const contentsInAllCells = {
-    html: '<table style = "width: 100%;">' +
-            '<tbody>' +
-              '<tr>' +
-                '<td>a1</td>' +
-                '<td>b1</td>' +
-              '</tr>' +
-              '<tr>' +
-                '<td>a2</td>' +
-                '<td>b2</td>' +
-              '</tr>' +
-              '<tr>' +
-                '<td>a3</td>' +
-                '<td>b3</td>' +
-              '</tr>' +
-            '</tbody>' +
-          '</table>'
+  const baseSettings = {
+    plugins: 'table',
+    base_url: '/project/tinymce/js/tinymce',
   };
 
-  const tableWithHeadings = {
-    html: '<table style = "width: 100%;">' +
-            '<tbody>' +
-              '<tr>' +
-                '<th>a1</th>' +
-                '<th>b1</th>' +
-              '</tr>' +
-              '<tr>' +
-                '<td>a1</td>' +
-                '<td>b1</td>' +
-              '</tr>' +
-              '<tr>' +
-                '<td>a2</td>' +
-                '<td>b2</td>' +
-              '</tr>' +
-            '</tbody>' +
-          '</table>'
-  };
+  const platform = PlatformDetection.detect();
 
-  const tableWithCaption = {
-    html: '<table style = "width: 100%;">' +
-            '<caption>alphabet</caption>' +
-            '<tbody>' +
-              '<tr>' +
-                '<td>a1</td>' +
-                '<td>b1</td>' +
-              '</tr>' +
-              '<tr>' +
-                '<td>a2</td>' +
-                '<td>b2</td>' +
-              '</tr>' +
-            '</tbody>' +
-          '</table>'
-  };
+  const getTableWidth = (editor: Editor) =>
+    SelectorFind.child<HTMLTableElement>(TinyDom.body(editor), 'table').map(Width.get).getOrDie();
 
-  const nestedTables = {
-    html: '<table style = "width: 100%;">' +
-            '<tbody>' +
-              '<tr>' +
-                '<td>a1' +
-                  '<table style = "width: 100%;">' +
-                  '<tbody>' +
-                    '<tr>' +
-                      '<td></td>' +
-                      '<td></td>' +
-                    '</tr>' +
-                    '<tr>' +
-                      '<td></td>' +
-                      '<td></td>' +
-                    '</tr>' +
-                  '</tbody>' +
-                  '</table>' +
-                '</td>' +
-                '<td>b1</td>' +
-              '</tr>' +
-              '<tr>' +
-                '<td>a2</td>' +
-                '<td>b2</td>' +
-              '</tr>' +
-            '</tbody>' +
-          '</table>'
+  const assertNewWidth = (editor: Editor, multiplier: number) => {
```

Comment:
This is doing much more than asserting a new width, so maybe `insertColsAndAssertNewWidth`?